<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matchmaking</class>
 <widget class="QWidget" name="matchmaking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="back">
       <property name="minimumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>⬅</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="pet_image">
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>175</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1600</width>
       <height>1600</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Picture Goes Here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PB_switchImg">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>161111</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 9px;
    background-color: #f0f0f0;
}</string>
     </property>
     <property name="text">
      <string>Switch Image</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="pet_name">
        <property name="font">
         <font>
          <family>Manjari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pet Name + Distance Goes Here</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="dislike">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>65</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 32px;
    background-color: #ea4c46;

}</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="neutral">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>65</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 32px;
    background-color: #fcf4a3;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="like">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>65</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Manjari</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 32px;
    background-color: #3ded97;
}</string>
       </property>
       <property name="text">
        <string>✓</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="request">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 25px;
    background-color: #ffb6c1;
}</string>
     </property>
     <property name="text">
      <string>Request Adoption!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
